recipes/python-swiftclient/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "python-swiftclient" %}
+{% set version = "3.5.0" %}
+{% set sha256 = "8d91368f99bf659d2889b01fa2a796e2b36c7397073dd365a7337624bc1b0836" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  fn: "{{ name }}-{{ version }}.tar.gz"
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pbr >=2.0.0
+    - pip
+  run:
+    - python
+    - pip     # needed for `pkg_resources` import
```

Comment:
It's very good to have the comment about `pkg_resources` being used, thanks!
But that should be `setuptools` since `pkg_resources` is officially part of `setuptools`.